apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ErrorReportingTest.cs

```diff
@@ -90,8 +89,7 @@ public async Task LogsMultipleExceptions()
             response = await requestTask4;
             Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
 
-            var errorEvents = s_polling.GetEvents(_testId, 4);
-            Assert.Equal(4, errorEvents.Count());
+            var errorEvents = ErrorEventEntryVerifiers.VerifyMany(s_polling.GetEvents(_testId, 4), 4, _testId);
```

Comment:
Likewise this could be `ErrorEventEntryVerifiers.VerifyMany(s_polling, _testId, 4)`